server/src/main/java/org/eclipse/openvsx/util/UrlUtil.java

```diff
@@ -138,12 +135,35 @@ public static String addQuery(String url, String... parameters) {
         return result.toString();
     }
 
+    public static HttpHeaders getForwardedHeaders() {
```

Comment:
How is this method related to URLs?

Reply:
I think It was added here because forwarded headers are handled below. But can moved to another place.